ignite/cmd/plugin.go

```diff
@@ -343,6 +347,112 @@ func NewPluginUpdate() *cobra.Command {
 	}
 }
 
+func NewPluginAdd() *cobra.Command {
+	cmdPluginAdd := &cobra.Command{
+		Use:  "add [path]",
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return fmt.Errorf("must provide a plugin path")
+			}
+			chain, err := newChainWithHomeFlags(cmd)
+			s := cliui.New(cliui.WithStdout(os.Stdout))
+
+			if err != nil {
+				return err
+			}
+			conf, err := chain.Config()
+			if err != nil {
+				return err
+			}
+
+			for _, p := range conf.Plugins {
+				if p.Path == args[0] {
+					return fmt.Errorf("cannot add duplicate plugin %s", args[0])
+				}
+			}
+
+			p := chainconfig.Plugin{
+				Path: args[0],
+			}
+
+			conf.Plugins = append(conf.Plugins, p)
+			s.Printf("🎉 %s added \n", args[0])
+			err = chainconfig.Save(*conf, chain.ConfigPath())
+
+			if err != nil {
+				return err
+			}
+
+			if flag, err := cmd.Flags().GetBool("load"); err == nil && flag {
+				chain, err = newChainWithHomeFlags(cmd)
```

Comment:
Do you really need to call it a second time ?

Reply:
In my findings with `config` instance returned from `chain.Config()` the pointer to the struct was not updated with new plugin instances. I found that passing the config to load without an explicit reload would not respect the newly added plugin. So I started thinking of this as a transactional flow where the config must be saved to disk and reloaded in order to respect the newly added plugin decleration.

Reply:
OK, anyway this will change with #3184, so I suggest we wait for its merge and then you will have to change how the config is loaded.

Reply:
have upgraded to the new plugin config and removed this duplicate `newChainWithHomeFlags`.